src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -265,6 +265,15 @@ public static class Popup
 			/// </summary>
 			public static bool UseNativePopup { get; set; } = true;
 #endif
+
+			/// <summary>
+			/// By default, light dismiss is disabled in UWP/WinUI unless
+			/// IsLightDismissEnabled is explicitly set to true.
+			/// In earlier versions of Uno Platform this property defaulted
```

Comment:
Minor grammar correction

```suggestion
			/// In earlier versions of Uno Platform, this property defaulted
```